lib/main.js

```diff
@@ -2,6 +2,7 @@ var assign = require('object-assign')
 var path = require('path')
 var BrowserWindow = require('browser-window')
 var wargs = require('./args')
+var v = (process.versions.electron || '').split('.').map(Number)
```

Comment:
This could be a good place for the new assignment magic in ES2015:

```js
let [major, minor, patch] = process.versions.electron.split('.').map(Number)
```